source/mge-metadata.json

```diff
@@ -0,0 +1,2991 @@
+        "schema": {
+            "attributes": [
+                "self"
+            ],
+            "category": "Layer",
+            "opname": "flatten"
+        }
+    },
+    {
+        "name": "sum",
+        "schema": {
+            "attributes": [
+                "self",
+                "axis",
+                "keepdims"
+            ],
+            "category": "Layer",
+            "opname": "sum"
+        }
+    },
+    {
+        "name": "prod",
+        "schema": {
+            "attributes": [
+                "self",
+                "axis",
+                "keepdims"
+            ],
+            "category": "Layer",
+            "opname": "prod"
+        }
+    },
+    {
+        "name": "min",
+        "schema": {
+            "attributes": [
+                "self",
+                "axis",
+                "keepdims"
+            ],
+            "category": "Layer",
+            "opname": "min"
+        }
+    },
+    {
+        "name": "max",
+        "schema": {
+            "attributes": [
+                "self",
+                "axis",
+                "keepdims"
+            ],
+            "category": "Layer",
+            "opname": "max"
+        }
+    },
+    {
+        "name": "mean",
+        "schema": {
+            "attributes": [
+                "self",
+                "axis",
+                "keepdims"
+            ],
+            "category": "Layer",
+            "opname": "mean"
+        }
+    },
+    {
+        "name": "__getitem__",
+        "schema": {
+            "attributes": [
+                "self",
+                "index"
+            ],
+            "category": "Layer",
+            "opname": "getitem"
+        }
+    },
+    {
+        "name": "__setitem__",
+        "schema": {
+            "attributes": [
+                "self",
+                "index",
+                "value"
+            ],
+            "category": "Layer",
+            "opname": "setitem"
+        }
+    },
+    {
+        "name": "megengine.module.adaptive_pooling.AdaptiveAvgPool2d",
+        "schema": {
+            "category": "Pool"
+        }
+    },
+    {
+        "name": "megengine.module.adaptive_pooling.AdaptiveMaxPool2d",
+        "schema": {
+            "category": "Pool"
+        }
+    },
+    {
+        "name": "megengine.module.pooling.AvgPool2d",
+        "schema": {
+            "category": "Pool"
+        }
+    },
+    {
+        "name": "megengine.module.batch_matmul_activation.BatchMatMulActivation",
+        "schema": {
+            "category": "Activation"
+        }
+    },
+    {
+        "name": "megengine.module.batchnorm.BatchNorm1d",
+        "schema": {
+            "category": "Normalization"
+        }
+    },
+    {
+        "name": "megengine.module.batchnorm.BatchNorm2d",
+        "schema": {
+            "category": "Normalization"
+        }
+    },
+    {
+        "name": "megengine.module.concat.Concat",
+        "schema": {
+            "category": "Layer"
```

Comment:
Fix category for multiple operators. For example: `Concat`, `getitem`, `floordiv`...